packages/client/src/constants/maps.js

```diff
@@ -16,6 +18,7 @@ export const COIN_TYPE_TO_META = {
     publicReceiverPath: { domain: "public", identifier: "flowTokenReceiver" },
     publicBalancePath: { domain: "public", identifier: "flowTokenBalance" },
     storageVaultPath: { domain: "storage", identifier: "flowTokenVault" },
+    icon: <FlowCoin />
```

Comment:
We should avoid adding component into the config file and opt in referring the icon with its path then dynamically import the icons where it's being used. I'm ok for leaving it as it is right now to get the rest of the work unblocked, I'll take a note for myself to refactor it later on [this ticket](https://linear.app/dappercollectives/issue/VSL-183)

Reply:
Why would placing a component into a map be an issue?

Reply:
1. Config files should handle the single responsibility of giving the caller directions of where to find things or what thing the caller should find, putting the component in here would be equivalent to fetching the component and then handing it to the caller, it creates unnecessary chain of dependencies.
2. Let's say we have 7 token svgs all configured in the map, and you want to render the flow icon, you will end up loading all 7 of the svgs in the memory when you only need 1

Reply:
Gotcha! Thanks!

Reply:
I understood that this will be addressed in [this ticket](https://linear.app/dappercollectives/issue/VSL-183)

Reply:
ah sorry missed that.